packages/cli/test/integration.test.ts

```diff
@@ -265,7 +342,11 @@ const createUser = async () => {
 
 const getConfigAuthPath = () => path.join(globalDir, 'auth.json');
 
-async function setupProject(process, projectName, overrides) {
+async function setupProject(
+  process: BoundChildProcess,
+  projectName: string,
+  overrides: { devCommand?: any; buildCommand?: any; outputDirectory?: any }
```

Comment:
These should probably be `string`

Reply:
Yes all 3 are strings

Reply:
Updated: [`5acb8bf` (#9672)](https://github.com/vercel/vercel/pull/9672/commits/5acb8bf06d37985ee09d6a001c38f7664e6f1732)